community/SDK-GOVERNANCE.md

```diff
@@ -140,8 +140,8 @@ The voting criteria are:
 
 ### Archive a project
 
-If a project is not following the criteria 1, 2, 4, 5, 6, 7, the community MAY
-decides to archive the project. Prior to archiving, the community SHOULD first
+If a project is not following the criteria 1, 2, 4, 5, 6, 7, the community may
+decides to archive the project. Prior to archiving, the community should first
```

Comment:
...the community may **decide** to archive...

Reply:
next PR